webapp/channels/src/components/channel_header_mobile/show_search_button/show_search_button.tsx

```diff
@@ -15,24 +13,24 @@ type Props = {
     actions: Actions;
 }
 
-export default class ShowSearchButton extends React.PureComponent<Props> {
-    handleClick = () => {
-        this.props.actions.openRHSSearch();
+const ShowSearchButton: React.FC<Props> = ({ actions }) => {
```

Comment:
nit: can you change it to fit more with the style that we are using in the examples? Instead of using the `React.FC<Props>` explicit type declaration, using `({ actions }: Props) => ` as the function definition, that would fit well with the type checking and is align in style with the other functional components.

Reply:
I understand what you want. I have done the necessary changes.